packages/edit-site/src/components/page-patterns/patterns-list.js

```diff
@@ -78,6 +78,16 @@ export default function PatternsList( { categoryId, type } ) {
 		}
 	);
 
```

Comment:
The [latest thinking on useEffect](https://react.dev/learn/you-might-not-need-an-effect) seems to be that it shouldn't be used for syncing internal state, this should be done in event handlers instead where possible, so switched to event handlers here.

Reply:
I believe you are correct it shouldn't have been in a useEffect, I was thinking straight when I quickly hacked this up last night.

Thanks for pushing the tweak 🙇

Reply:
I missed this. Yes, I believe updating in event handlers is the correct way! I was doing the same in my exploration PR too.